tailwind.config.js

```diff
@@ -1,4 +1,16 @@
 'use strict';
+
+const except = (exception, object) => {
+	let output = {};
+
+	Object.keys(object).forEach(key => {
```

Comment:
.map instead of a external let and .forEach ?

Reply:
also `(key) => ...` for code style 😁

Reply:
> .map instead of a external let and .forEach ?

Hum.. map would only give me another array but I need an object as the output here. 🤔

Reply:
nah laisse faire le .map jme mélange avec les arrays. mais le param key reste valide !

Reply:
ouep, incoming! 🥇